.docker/Dockerfile.rhel

```diff
@@ -0,0 +1,44 @@
+FROM registry.access.redhat.com/rhscl/nodejs-8-rhel7
+
+ENV RC_VERSION 0.63.1
```

Comment:
this files needs to be added to [this list](https://github.com/RocketChat/Rocket.Chat/blob/develop/.scripts/set-version.js#L21) so this line can be easily updated via script

Reply:
done